packit_service/worker/helpers/testing_farm.py

```diff
@@ -1459,6 +1459,49 @@ def _payload_installability(self, distro: str, compose: str) -> dict:
             },
         }
 
+    @implements_fedora_ci_test(
+        "custom",
+        skipif=lambda service_config, project, metadata: not project.get_file_content(
+            path=".fmf/version", ref=metadata.commit_sha
+        ),
+    )
+    def _payload_custom(self, distro: str, compose: str) -> dict:
+        return {
+            "api_key": self.service_config.testing_farm_secret,
+            "test": {
+                "tmt": {
+                    "url": self.project.get_pr(self.metadata.pr_id).source_project.get_web_url(),
+                    "ref": self.metadata.commit_sha,
+                },
+            },
+            "environments": [
+                {
+                    "arch": "x86_64",
+                    "os": {"compose": compose},
+                    "variables": {
+                        "KOJI_TASK_ID": self.koji_build.task_id,
+                    },
+                    "artifacts": [
+                        {"id": self.koji_build.task_id, "type": "fedora-koji-build"},
+                    ],
+                    "tmt": {
+                        "context": {
+                            "distro": distro,
+                            "arch": "x86_64",
+                            "trigger": "commit",
+                            "initiator": "packit-ci",
```

Comment:
As for the question in the description, I'm not sure if tmt/TF validates the `initiator` value, but IMO this should be eventually switched to `fedora-ci` and also Packit is CI anyways, so… maybe `packit-fedora-ci` would be good substitute for now

Reply:
I don't think it validates it, but some test plans may rely on it, I think we have even some such condition in specfile (not to run certain tests on dist-git PRs, only in upstream CI). `packit-fedora-ci` sounds good.